build.gradle

```diff
@@ -799,33 +799,8 @@ def checkstyleConfigProperties(configFileName) {
   [importControlFile: "$configFileName"]
 }
 
-// Aggregates all jacoco results into the root project directory
```

Comment:
Where would one use this functionality and why don't we need it?

Reply:
Ok, that sounds good. I think the next step to this PR would be to actually enable the jacoco plugin in jenkins, I'll look into how can we do that.